lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -2169,6 +2167,50 @@ public function getUpdatedConfig()
         return $configJson;
     }
 
+    /**
+     * Updates attributeLayers configuration object with layers export permissions information.
+     *
+     * @param object   $configJson Project configuration object
+     * @param string[] $userGroups User groups
+     */
+    public function setLayersExportPermissions(&$configJson, $userGroups)
+    {
+        if (property_exists($configJson, 'attributeLayers')) {
```

Comment:
Can you switch to `getPropertyOrDefault` ? It's the purpose of this method, to avoid writing twice the key.

And all other occurrences in the code you are adding `export_enabled` and `export_allowed_groups`.

Side note, but I'm in favor of exiting a function as soon as possible, when conditions are not met. (to avoid to wonder if some code are executed below a lonnnng block of code. If the property doesn't exist, I usually return from the function. I find this code clearer and more secure.
And it also deindent your code, so easier to keep code on the left and keep 120 chars.